README.md

```diff
@@ -1,6 +1,6 @@
 # Prometheus Python Client
 
-The official Python 2 and 3 client for [Prometheus](http://prometheus.io).
+The official Python 3 client for [Prometheus](https://prometheus.io).
```

Comment:
Do you think we could just call this `The official Python client` rather than specifying version?

Reply:
Good idea, will update.